static-site/_includes/head.html

```diff
@@ -0,0 +1,14 @@
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    {%- seo -%}
+    <link rel="stylesheet" href="{{ "/assets/main.css" | relative_url }}">
+    {%- feed_meta -%}
+    {% if page.url == "/swagger.html" %}
```

Comment:
This is annoying, but the only way to override the head content is by copying the entire file from the theme we use ([minima](https://github.com/jekyll/minima/blob/v2.5.2/_includes/head.html)) and changing the parts you need. Furthermore, if we only want to customize the head for a single page, it seems like we need to use a conditional like this.